### PR TITLE
added new directory creation for dataset create

### PR DESCRIPTION
**Pull request by ExtinctWolf83**

### CODE REVIEW

File: deploifai/dataset/create.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import click
 import click_spinner
 from PyInquirer import prompt
+import os
+from time import sleep
```
Comment (98sean98) [inline deploifai/dataset/create.py:5]: shouldn't require the `time` module at all

File: deploifai/dataset/create.py [hunk 2]
```diff
@@ -134,3 +136,20 @@ def create(context: DeploifaiContextObj):
         azure_account = cloud_data["cloudProviderYodaConfig"]["azureConfig"]["storageAccount"]
         click.secho("account-name: {}".format(azure_account))
         click.secho("container-name: {}".format(cloud_name))
+
+    # create a dataset directory locally, along with dataset.cfg file within
+    try:
+        os.mkdir(storage_name)
+    except OSError:
+        click.secho("An error occured when creating the dataset locally", fg="red")
```
Comment (98sean98) [inline deploifai/dataset/create.py:144]: should probably add another echo after this error message saying `But the dataset has been created on Deploifai`

File: deploifai/utilities/config/dataset_config.py [hunk 3]
```diff
@@ -38,13 +38,17 @@ def __init__(self, message):
         super(DatasetNotInitialisedError, self).__init__(message)
 
 
-def create_config_files():
+def create_config_files(new_dataset_dir= None):
```
Comment (98sean98) [inline deploifai/utilities/config/dataset_config.py:41]: let's do `new_dataset_dir:str = None`

**Review by 98sean98 — CHANGES_REQUESTED** [review]
Just minor semantic changes required.

File: deploifai/dataset/create.py (hunk 1 above)
Comment (ExtinctWolf83) [reply deploifai/dataset/create.py:5]: its used in the spinner part of the code

Comment (98sean98) [reply deploifai/dataset/create.py:5]: oh ok, my mistake

**Review by 98sean98 — APPROVED** [review]

**Pull request merged**